src/test_helpers/component.cljs

```diff
@@ -53,8 +53,10 @@
   (rtl/render (reagent/as-element component)))
 
 (defn render-with-theme-provider
```

Comment:
@ilmotta - I believe your suggestion was to update `render` method instead, right??